packit/config/package_config.py

```diff
@@ -271,6 +291,12 @@ def find_remote_package_config(
             f"for the {project.full_repo_name!r} repository."
         )
         return None
+    except GithubAPIException as ex:
```

Comment:
Can this occur only for github? What about the other forges?

Reply:
I'll have look, currently I'm considering burning the flexmock down…